test/parallel/test-whatwg-webstreams-transfer.js

```diff
@@ -464,12 +464,17 @@ const theData = 'hello';
       tracker.verify();
     });
 
+    // We create an interval to keep the event loop alive while
```

Comment:
Something seems off if this is needed. Maybe I'm misunderstanding the test though.